packages/searchable-dropdown/src/provider/controller.ts

```diff
@@ -49,8 +48,8 @@ export class SearchableDropdownController implements ReactiveController {
         } else {
           result = await this.resolver.searchQuery(qs);
         }
-        // set isSelected on result items
-        this.result = this.mutateResult(result);
+
+        this.result = result;
```

Comment:
Previously `mutateResult` would go through and set `isSelected` on the result. Now that doesn't happen.